<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="simulationTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="nameLabel">
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="nameEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="usedByLabel">
               <property name="text">
                <string>Used By:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="usedByEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>&amp;Plugins</string>
    </property>
    <addaction name="actionView"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionAdd_Model"/>
    <addaction name="actionRemove_Model"/>
    <addaction name="actionExport_Model"/>
    <addaction name="actionImport_Model"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuModel"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Model"/>
   <addaction name="actionRemove_Model"/>
   <addaction name="actionExport_Model"/>
   <addaction name="actionImport_Model"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSimulation">
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="theme" resource="theme.qrc">
     <normaloff>:/icons/new</normaloff>:/icons/new</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="theme" resource="theme.qrc">
     <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="theme" resource="theme.qrc">
     <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>&amp;View</string>
   </property>
  </action>
  <action name="actionAdd_Model">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="theme.qrc">
     <normaloff>:/icons/model-add</normaloff>:/icons/model-add</iconset>
   </property>
   <property name="text">
    <string>Add Model</string>
   </property>
   <property name="toolTip">
    <string>Add Model</string>
   </property>
  </action>
  <action name="actionRemove_Model">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="theme.qrc">
     <normaloff>:/icons/model-remove</normaloff>:/icons/model-remove</iconset>
   </property>
   <property name="text">
    <string>Remove Model</string>
   </property>
   <property name="toolTip">
    <string>Remove Model</string>
   </property>
  </action>
  <action name="actionExport_Model">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="theme.qrc">
     <normaloff>:/icons/model-export</normaloff>:/icons/model-export</iconset>
   </property>
   <property name="text">
    <string>Export Model</string>
   </property>
  </action>
  <action name="actionImport_Model">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="theme.qrc">
     <normaloff>:/icons/model-import</normaloff>:/icons/model-import</iconset>
   </property>
   <property name="text">
    <string>Import Model</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="theme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionView</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewPlugins()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>newSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>viewPlugins()</slot>
  <slot>newSimulation()</slot>
  <slot>openSimulation()</slot>
  <slot>saveSimulation()</slot>
 </slots>
</ui>
